docker/Dockerfile

```
@@ -0,0 +1,36 @@
+# First stage: build the application
+FROM golang:1.13.0-alpine AS builder
```

Can't we update this? :smile: 

```suggestion
FROM golang:1.14-alpine AS builder
```

---

Done.